easybuild/easyconfigs/b/BLAST+/BLAST+-2.7.1_boost_backport.patch

```diff
@@ -0,0 +1,157 @@
+backporting Boost 1.66 support from BLAST+ 2.8 alpha
```

Comment:
@Serpens Do you have an upstream reference (URL) for this patch? Please include it...

Reply:
@boegel I got the idea from this discussion https://aur.archlinux.org/packages/blast%2B/ but I did it on my own, so no upstream reference. Should I find who and when added Boost 1.66 support in NCBI's SVN?

Reply:
If the patch is based on an idea and you created it yourself, just including yourself as an author is sufficient imho.

So, let's just get rid of the easyconfig for Boost 1.63.0?